comictaggerlib/ctoptions/talker_plugins.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+import logging
+from collections.abc import Mapping
+
+import settngs
+
+from comictalker.talkerbase import ComicTalker
+
+logger = logging.getLogger(__name__)
+
+
+def register_talker_settings(parser: settngs.Manager, plugins: Mapping[str, ComicTalker]) -> None:
+    for talker_name, talker in plugins.items():
+        try:
+            parser.add_group(talker_name, talker.register_settings, False)
```

Comment:
I want all of these groups to have `talker_` as a prefix in the group name. I don't want it to be possible to have a name conflict with a ComicTagger group. If two talkers have a conflict they can deal with it themselves.